Assets/Code/Scripts/LevelManagement/CheckpointController.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CheckpointController : MonoBehaviour
+{
+    private GameManager _gameManager;
+    /*
+     * CheckpointController checks if the player has collided with a checkpoint.
+     * If a new checkpoint is hit, then it'll be overwritten.
+     */
```

Comment:
If we want to be picky, here [are the docstrings guidelines for C#.](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) So, for example, this would be as follows:

```suggestion
    /// <summary>
    /// CheckpointController checks if the player has collided with a checkpoint.
    /// If a new checkpoint is hit, then it'll be overwritten.
    /// </summary>
```